### PR TITLE
add feature toggle for request infos

### PR DESCRIPTION
**Pull request by CordlessWool**
# Description

Add Feature flag for logging request infos.

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-8447

## Deployment

FEATURE_LOG_REQUEST default not set

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by tofixx — APPROVED** [review]
go for it, everything else can be added later.

File: config/default.schema.json [hunk 1]
```diff
@@ -66,6 +66,11 @@
 			"default": false,
 			"description": "If enabled, firstLogin page is not shown."
 		},
+		"FEATURE_LOG_REQUEST": {
```
Comment (CeEv) [inline config/default.schema.json:69]: Please enable it for development as default. For production default false is ok.

**Review by CeEv — CHANGES_REQUESTED** [review]

**Review by CeEv — DISMISSED** [review]
Please fix by merge into develop that request info is show in development mode, over config/develop.json.

**Comment by CeEv** [comment]
I see that changelog and versions are is missing.

File: config/default.schema.json (hunk 1 above)
Comment (Ojself) [reply config/default.schema.json:64]: fixed

**Pull request merged**